docs/hub/repositories.md

```diff
@@ -0,0 +1,212 @@
+
+```bash
+python -m pip install huggingface_hub
+huggingface-cli login
+```
+
+The content in the **Getting Started** section of this document is also available as a video!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rkCly_cbMBk" title="Managing a repo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Creating a repository
+
+Using the Hub's web interface you can easily create repositories, add files (even large ones!), explore models, visualize diffs, and much more. There are three kinds of repositories on the Hub, and in this guide we'll be creating a **model repository** for demonstration purposes. For information on creating and managing models, datasets, and Spaces, refer to their respective documentation.
+
+1. To create a new repository, visit [huggingface.co/new](http://huggingface.co/new):
+
+![/docs/assets/hub/new_repo.png](/docs/assets/hub/new_repo.png)
+
+2. First, specify the owner of the repository: this can be either you or any of the organizations you’re affiliated with. 
+
+3. Next, enter your model’s name. This will also be the name of the repository. Finally, you can specify whether you want your model to be public or private.
+
+You can leave the *License* field blank for now. To learn about licenses, visit the **Licenses** (TODO: LINK TO LICENSES) section of this document.
+
+After creating your model repository, you should see a page like this:
+
+![/docs/assets/hub/empty_repo.png](/docs/assets/hub/empty_repo.png)
+
+Note that the Hub prompts you to create a *Model Card*, which you can learn about in the **Model Cards documentation** (TODO: LINK). Including a Model Card in your model repo is best practice, but since we're only making a test repo at the moment we can skip this.
+
+
+### Cloning repositories
+
+Downloading repositories to your local machine is called *cloning*. You can use the following commands to load the repo that we made and navigate to it:
+```bash
+git clone https://huggingface.co/<your-username>/<your-model-id>
+cd <your-model-id>
+```
+
+Then, *within the local repo*, you'll need to run the following to set up Git LFS:
+```bash
+git lfs install
+```
+
+### Adding files to a repository
+
+Now's the time 🔥 You can add any files you want to the repository!
+
+You can do this with the usual Git workflow:
+
+```bash
+# Create any files you like! Then...
+git add .
+git commit -m "First model version"  # You can choose any descriptive message
+git push
+```
+
+And we're done! You can check your repository on Hugging Face with all the recently added files. For example, in the screenshot below the user added a number of files. Note that one of the files in this example has a size of `413.3 MB`, so the repo would use Git LFS to track it.
+
+![/docs/assets/hub/repo_with_files.png](/docs/assets/hub/repo_with_files.png)
```

Comment:
Is it just my browser or are the file names in the image weirdly truncated? Also, I think we should be adding documentation-related images to this Hub [repo](https://huggingface.co/datasets/huggingface/documentation-images) and referencing them from there, instead of adding them to the GitHub repo :)

Reply:
Nope, not just you! This is one of the old images and I just didn't re-do it. Totally, I can move this kind of stuff over to the Hub repo!

Reply:
I'm going to move the images over to the Hub repo, but I'm just running into some issues (https://huggingface.slack.com/archives/C01BWJU0YKW/p1649820110948279). Will do that as soon as I've figured the issue out!